CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
+
+All notable changes for the LINZ BDE schema are documented in this file.
+
+## 2.0.0 - 2020-xx-xx
+- QGIS3 Porting - PLACEHOLDER, STILL TO BE RELEASED
```

Comment:
It would be good to categorise the changes to make it a bit more understandable and so anyone reading it can find the changes they care about the most. I always like the classic:

### Features
### Bugs
### Docs

QGIS3 Porting would be a feature I guess.